consensus/src/quorum_store/proof_coordinator.rs

```diff
@@ -72,34 +62,20 @@ impl IncrementalProofState {
         Ok(())
     }
 
-    fn ready(&self, validator_verifier: &ValidatorVerifier, my_peer_id: PeerId) -> bool {
-        self.aggregated_signature.contains_key(&my_peer_id)
-            && validator_verifier
-                .check_voting_power(self.aggregated_signature.keys())
-                .is_ok()
+    fn ready(&self, validator_verifier: &ValidatorVerifier) -> bool {
+        validator_verifier
```

Comment:
I still think we should log something if signature doesn't contain own key. It means we are processing network requests faster than own, and smt is def wrong.

Reply:
we can log something but it shouldn't prevent the quorum from created